.github/workflows/audit-bootnodes.yml

```diff
@@ -0,0 +1,77 @@
+name: Audit Bootnodes
+
+on:
+
+  schedule:
+    - cron: '0 6 * * *'
+
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'params/bootnode*'
```

Comment:
My intention here is for the workflow to trigger with a `pull_request` on `master` AND where a file matching the pattern under `paths` is modified. (Not OR.)

Reply:
I've run some tests to experiment and prove that my logic about the AND vs. OR is correct, and it appears to be: https://github.com/meowsbits/ci-tests/actions.